src/Twig/ClassViewFactory.php

```diff
@@ -0,0 +1,202 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace PhpDocumentGenerator\Twig;
+
+use PhpDocumentGenerator\Parser\ClassParser;
+use PhpDocumentGenerator\Parser\MethodParser;
+use PhpDocumentGenerator\Parser\ParameterParser;
+use PhpDocumentGenerator\Parser\PropertyParser;
+use PhpDocumentGenerator\Twig\View\ClassView;
+use PhpDocumentGenerator\Twig\View\ConstantView;
+use PhpDocumentGenerator\Twig\View\MethodParameterView;
+use PhpDocumentGenerator\Twig\View\MethodView;
+use PhpDocumentGenerator\Twig\View\PropertyView;
+use PhpDocumentGenerator\Twig\View\TypeView;
+use PHPStan\PhpDocParser\Ast\Type\IntersectionTypeNode;
+use PHPStan\PhpDocParser\Ast\Type\TypeNode;
+use PHPStan\PhpDocParser\Ast\Type\UnionTypeNode;
+use ReflectionClass;
+use ReflectionMethod;
+use ReflectionNamedType;
+use ReflectionParameter;
+use ReflectionProperty;
+
+final class ClassViewFactory
```

Comment:
This class is complex, could it be splitted in other classes, such as MethodViewFactory, ConstantViewFactory, etc.?